code-studio/scripts/build-js.js

```diff
@@ -77,6 +77,16 @@ Promise.all([
     commonFile: 'plc'
   })),
 
+  // Bundle for assessment JS
+  build_commands.bundle(_.extend({}, defaultOptions, {
+    filenames: [
+      'assessments/levelGroup.js',
+      'assessments/multi.js',
+      'assessments/textMatch.js'
+    ],
+    commonFile: 'assessments'
```

Comment:
Why not just make this part of the code-studio bundle, like `levels/contract_match.jsx` and `levels/widget.js`?  Do these include a large amount of common code (e.g. a particular library) that we don't want to load everywhere?